signing/signing.go

```diff
@@ -122,10 +140,12 @@ func (s *Signing) ProcessSignature(jsonPayload []byte, repoName, repoRef string)
 		Result      string `json:"result"`
 		Branch      string `json:"branch"`
 		AccessToken string `json:"accessToken"`
+		TlogIndex   int64  `json:"tlogIndex"`
```

Comment:
This is `logIndex` in cosign.bundle but `tlogIndex` here, is that intentional?

Reply:
I tried to provide additional context as a POST field. When reading `logIndex` in `cosign.bundle`, it's easy to know which log index it's referring to. When sending it to the webapp, there's no cosign context clue, so I had added `tlogIndex`.